api/py/ai/chronon/repo/compile.py

```diff
@@ -99,7 +99,7 @@ def extract_and_convert(chronon_root, input_path, output_root, debug, force_over
     else:
         raise Exception(f"Input Path: {full_input_path}, isn't a file or a folder")
     validator = ChrononRepoValidator(chronon_root_path, output_root, log_level=log_level)
-    extra_online_group_bys = {}
+    extra_online_bsd_group_bys = {}
```

Comment:
hmm, i don't know how i feel about `bsd`. how about `gb_backfill_enabled_group_bys`, and `extra_online_or_gb_backfill_enabled_group_bys`. I'd rather be verbose.